samples/Config.Options.Snapshot.Web/Config.Options.Snapshot.Web.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
It doesn't matter for samples and src projects. We only require it for test projects.